Gemfile.lock

```diff
@@ -692,7 +692,7 @@ GEM
     uri_template (0.7.0)
     utf8-cleaner (0.2.5)
       activesupport
-    vcr (3.0.3)
+    vcr (5.0.0)
```

Comment:
This seems errant, though if we merge the VCR PR #3583 first then we'll be fine.